components/home/HomeHeaderSection.tsx

```diff
@@ -0,0 +1,31 @@
+const HomeHeaderSection = () => {
+  return (
+    <section className="nhsuk-hero">
+      <div className="nhsuk-width-container nhsuk-hero--border app-width-container">
+        <div className="nhsuk-grid-row">
+          <div className="nhsuk-grid-column-two-thirds">
```

Comment:
`nhsuk-grid-column-three-thirds` stretches this out to fill the width 🤷

Reply:
three-thirds goes slightly outside the container. Could remove the grid css altogether though if we want full container width?... You've twisted my arm @ReubenRobertsHEE  I'll push the change with full width header.  😀  
![image](https://user-images.githubusercontent.com/42062738/220980759-85e7e357-678d-46d6-92de-ccf0ef147e74.png)